### DISCUSSION TITLE
How to get the full line highlighting like in the example images?

### DISCUSSION THREAD

**Discussion by damccull** [folke/todo-comments.nvim#278] [Q&A]
It's annoying that the images on the home page look so pretty, with fancy colored symbols, full lines highlighted, even across multiple lines...
![image](https://user-images.githubusercontent.com/292349/118135272-ad21e980-b3b7-11eb-881c-e45a4a3d6192.png)

...and then the default config gives this (color choices not relevant, they're from my theme, obviously):
![image](https://github.com/folke/todo-comments.nvim/assets/401104/28f7b70c-daac-4f40-b18d-ee81bf5c8d25)

How do I get those fancy icons in the left line? I am running gitsigns as well...is that overwriting it maybe?

How do I get the full-line text highlights? The multi-line highlights?

**Comment by folke** [comment]
You forgot the colon `-- HACK: foo`

**Reply by damccull** [reply]
Hahaha! I see. That _is_ a big difference. Thank you!
